ui-tests/hidden.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Hypermedia Client Testing</title>
+    <link rel="stylesheet" type="text/css" href="default.css" />
+    <script src="client.min.js"></script>
+  </head>
+  <body>
+    <div id="demo-app" data-endpoint="https://cignium-lm-staging.azurewebsites.net/run/staging/regression/prod/ui-disabled-hidden"></div>
```

Comment:
Do we want to put `https://cignium-lm-staging.azurewebsites.net/run/staging/regression/prod/` in an environment variable?